posthog/api/__init__.py

```diff
@@ -68,6 +69,9 @@ def api_not_found(request):
 organizations_router.register(
     r"invites", organization_invite.OrganizationInviteViewSet, "organization_invites", ["organization_id"],
 )
+organizations_router.register(
```

Comment:
FYI @Twixes this PR includes the test cases for https://github.com/PostHog/posthog/pull/9004